src/main/external-resources/renderers/Microsoft-XboxOne.conf

```diff
@@ -24,10 +24,10 @@ LoadingPriority = 1
 
 SeekByTime = exclusive
 TranscodeVideo = MPEGTS-H264-AC3
-MediaInfo = true
 ThumbnailWidth = 256
 ThumbnailHeight = 256
 HalveBitrate = true
+MediaInfo = true
 
```

Comment:
@Sami32 If you're not making a change to this file, it's better to exclude the file from the commit IMO. It will make reading the history for the file easier. In this case though, you should remove the Thumbnail properties which will lead to the file being included anyway.